tests/utils.js

```diff
@@ -492,6 +492,13 @@ define([
                 }
                 mug.p[name] = val;
             });
+
+            // Open all sections
```

Comment:
I'm not excited about this as it means that most tests are run from a non-default setup. With that said, I wouldn't volunteer to go through each test to fix this either